sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -523,5 +523,14 @@ public async ValueTask DisposeAsync()
                 }
             }
         }
+
+        internal class Progress : IProgress<long>
```

Comment:
consider moving this into its own file and calling it something like TestProgress

Reply:
fixed.